src/app/retrospective/sprint-task-summary/sprint-task-summary.component.ts

```diff
@@ -642,26 +672,29 @@ export class SprintTaskSummaryComponent implements OnInit, OnChanges, OnDestroy
             dialogRef.afterClosed().takeUntil(this.destroy$).subscribe(result => {
                 if (result) {
                     currentDoneAt = sprintTaskSummaryData.DoneAt;
+                    currentResolution = sprintTaskSummaryData.Resolution;
                     sprintTaskSummaryData.DoneAt = null;
+                    sprintTaskSummaryData.Resolution = null;
                     params.node.setData(sprintTaskSummaryData);
-                    // Refresh the Mark Done/Undone cell to reflect the change in the 'Done' icon
+                    // // Refresh the Mark Done/Undone cell to reflect the change in the 'Done' icon
                     params.refreshCell({ suppressFlash: false, newData: false, forceRefresh: true });
                     this.retrospectiveService.markSprintTaskUnDone(this.retrospectiveID, this.sprintID, sprintTaskSummaryData.ID)
                         .takeUntil(this.destroy$)
                         .subscribe(
                             response => {
                                 const sprintTaskSummary = response.data;
                                 params.node.setData(sprintTaskSummary);
-                                // Refresh the Mark Done/Undone cell to reflect the change in the 'Done' icon
+                                // // Refresh the Mark Done/Undone cell to reflect the change in the 'Done' icon
```

Comment:
Same here and look for other places as well.

Reply:
done